whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -32,6 +33,7 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
```

Comment:
Can you change the comment on top of class as well

Reply:
Done